### ISSUE TITLE
Import synchro simulation

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#8740]
Apparently there is a csv format that describes the network. The attached example was provided by 

Yiran Zhang
Graduate Student
intelligent Urban Transportation Systems (iUTS) Lab
Department of CEE 
University of Washington
Email: yiranz94@uw.edu

[network_test_synchro.csv.zip](https://github.com/eclipse/sumo/files/6621577/network_test_synchro.csv.zip)

**Comment by namdre** [comment]
@mschrader15 are you aware of anyone working on something like this?

**Comment by mschrader15** [comment]
Nope, I've never seen this file format. I will ask around the CE department here though!

**Comment by namdre** [comment]
It seems to be somewhat official: Universal Traffic Data Format (UTDF): https://ops.fhwa.dot.gov/arterial_mgmt/rpt/sig_tim_proc/sect_3.htm

**Comment by namdre** [comment]
> Universal Traffic Data Format (UTDF) - Not Open Enough
Trafficware created the UTDF to import and export data to and from its Synchro traffic engineering software (14). Synchro and the format are widely used by traffic engineers and the format has seen support from a variety of other open source projects such as DTALite (15). However widely used, it is not publically managed or even documented and is thus not an “open” format in the common use of the term.
https://fast-trips.mtc.ca.gov/library/TRB17-OpenData.pdf

**Comment by namdre** [comment]
There does seem to be some open source code to read this stuff: https://code.google.com/archive/p/nexta/

**Comment by mschrader15** [comment]
Synchro is a commonly used tool so it probably makes sense to build a tool to convert it into NEMA configurations.

**Comment by mschrader15** [comment]
I'm meeting with @qichaow next week to discuss #11931 . Will add this to the agenda

**Comment by mschrader15** [comment]
We both think it is a good idea, but neither @qichaow or I have the bandwidth for it atm. @qichaow will ask around for interest.

**Comment by mschrader15** [comment]
Re-discovered this thread after the conference this year.


Here is the codebase that @Yiran6 presented at SUMO UC: https://github.com/Yiran6/Synchro2SUMO/blob/master/Sumo_transit_traffic_signal.ipynb
